helm/ph-ee-engine/connector/templates/secret.yaml

```diff
@@ -1,8 +1,10 @@
+{{- if .Values.enabled -}}
 apiVersion: {{ .Values.secret.apiversion }}
```

Comment:
this is not needed, this service can re-use the bulk processor AWS secret

Reply:
This will be addressed later as backlog